res/Sectors/M1105/Avereguar.txt

```diff
@@ -0,0 +1,339 @@
+# Generated by https://travellermap.com
```

Comment:
Avereguar sector has only two subsector capitals, and should have 5-8

Reply:
See above - added capitals